services/shhext/api.go

```diff
@@ -6,14 +6,16 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	whisper "github.com/ethereum/go-ethereum/whisper/whisperv6"
+	"github.com/status-im/status-go/services/shhext/dedup"
 )
 
 // NewPublicAPI returns instance of the public API.
-func NewPublicAPI(w *whisper.Whisper, tracker *tracker) *PublicAPI {
+func NewPublicAPI(w *whisper.Whisper, tracker *tracker, deduplicator *dedup.Deduplicator) *PublicAPI {
 	return &PublicAPI{
 		w:         w,
 		publicAPI: whisper.NewPublicWhisperAPI(w),
 		tracker:   tracker,
+		d:         deduplicator,
```

Comment:
Already had been a bit unhappy with `w` opposite to `tracker`. Now again `d`. Good opportunity to harmonize it?

Reply:
oh, forgot to rename it, thanks! 😅